doc/reference_output.xml

```diff
@@ -928,17 +927,28 @@ SELECT (ST_AsLatLonText('POINT (-302.2342342 -792.32498)'));
 				<funcdef>text <function>ST_AsText</function></funcdef>
 				<paramdef><type>geometry </type> <parameter>g1</parameter></paramdef>
 			  </funcprototype>
+			  <funcprototype>
+				<funcdef>text <function>ST_AsText</function></funcdef>
+				<paramdef><type>geometry </type> <parameter>g1</parameter></paramdef>
+				<paramdef choice="opt"><type>integer </type> <parameter>precision=15</parameter></paramdef>
+			  </funcprototype>
+			  <funcprototype>
+				<funcdef>text <function>ST_AsText</function></funcdef>
+				<paramdef><type>geography </type> <parameter>g1</parameter></paramdef>
+			  </funcprototype>
 			  <funcprototype>
 				<funcdef>text <function>ST_AsText</function></funcdef>
 				<paramdef><type>geography </type> <parameter>g1</parameter></paramdef>
+				<paramdef choice="opt"><type>integer </type> <parameter>precision=15</parameter></paramdef>
 			  </funcprototype>
 			</funcsynopsis>
 		  </refsynopsisdiv>
 
 		  <refsection>
 			<title>Description</title>
 
-			<para>Returns the Well-Known Text representation of the geometry/geography.</para>
+			<para>Returns the Well-Known Text representation of the geometry/geography. Optional argument may be used to reduce the maximum number
```

Comment:
Consider defaulting to 18, because people use WKT as an archive and storage format (even though the docs warn them not to).

Reply:
Or tap into `extra_float_digits` for consistency with `pg_dump` etc

Reply:
See [Trac#407](https://trac.osgeo.org/postgis/ticket/407) for the discussion of `extra_float_digits`.